src/VisualStudio/Core/Def/Implementation/TableDataSource/AbstractTableEntriesSnapshot.cs

```diff
@@ -202,14 +202,29 @@ protected string GetFileName(string original, string mapped)
 
         private string Combine(string path1, string path2)
         {
-            if (FilePathUtilities.TryCombine(path1, path2, out var result))
+            if (TryCombine(path1, path2, out var result))
             {
                 return result;
             }
 
             return string.Empty;
         }
 
+        public static bool TryCombine(string path1, string path2, out string result)
+        {
+            try
+            {
+                // don't throw exception when either path1 or path2 contains illegal path char
+                result = System.IO.Path.Combine(path1, path2);
```

Comment:
I moved the Try version into the code that was calling it.. as it was the only place it was used.  I'm not sure why it was using Path.Combine instead of the utility helper.  I kept it in case there was a difference that the feature depended on.